src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -396,8 +409,19 @@ public DeletePageObject deletePage() {
     return new DeletePageObject(driver);
   }
 
-  public String getBannerNotificationText() {
-    return bannerNotification.getText();
+  public List<Notification> getNotifications(){
+    List<Notification> notificationList = new ArrayList<>();
+    for (WebElement notificationElement : notificationElements){
+      Notification notification = new Notification(driver, notificationElement);
+      notificationList.add(notification);
+    }
+    return notificationList;
+  }
+
+  public List<Notification> getNotifications(NotificationType notificationType){
```

Comment:
Just a suggestion to rename it `getNotificationsByType`, as there are to very similar methods